src/main/java/com/rarchives/ripme/ripper/rippers/DeviantartRipper.java

```diff
@@ -344,7 +354,7 @@ public String getGID(URL url) throws MalformedURLException {
 		Matcher artistM = artistP.matcher(s);
 		if (s.endsWith("?catpath=/")) {
 			albumname = "all";
-		} else if (s.endsWith("/favourites/") || s.endsWith("/gallery/")) {
+		} else if (s.endsWith("/favourites/") || s.endsWith("/gallery/") || s.endsWith("/gallery") || s.endsWith("/favourites")) { //added andings without trailing / because of https://github.com/RipMeApp/ripme/issues/1303
```

Comment:
Shouldn't you just have the ones without a trailing slash since you'll match anyway ?

Reply:
I don't think so if I understand you correctly.
Deviantart redirects e.g. https://www.deviantart.com/apofiss/gallery to https://www.deviantart.com/apofiss/gallery/

so in a normal use case you would never have something similiar to the first URL. I just added theses because of the issue mentioned [here](https://github.com/RipMeApp/ripme/issues/1303). 
It might be a good idea to look into Regex again to replace them and make it look cleaner. 🤔